topics/icrc_28_trusted_origins.md

```diff
@@ -16,25 +17,37 @@
 
 This standard describes how a canister can indicate that a relying party (an entity that relies on the canister for certain functions or services) is trusted.
 
-Asset or other canisters that are meant to be composed upon by many parties in the ecosystem, such as ICRC-1 or ICRC-7 asset canisters, 
-**MUST** not implement ICRC-28 because it would privilege certain parties in a context where privilege should not exist.
+Canisters that manage tradable assets or are otherwise meant to be composed upon by distinct parties in the ecosystem (e.g. ICRC-1 or ICRC-7 canisters), 
+**MUST** not implement ICRC-28: ICRC-28 privileges the listed entities to potentially act independently on behalf of the signer, which is a security risk in the context of tradable assets and shared infrastructure.
 
-A trusted relying party carries certain privileges, like for example the ability to request Account Delegations as per ICRC-34.
+A trusted relying party carries certain privileges, like for example the ability to request Account Delegations as per [ICRC-34](./icrc_34_delegation.md).
 
 This standard assumes dapp developer responsibility for how a canister should:
 - Ensure that all entries are free from malicious intent.
-- Handle entries that become malicious over time.
+- Handle entries of entities that become malicious over time.
 
-## Method
+## Methods
 
 ### icrc28_trusted_origins
 
-Returns the canister's list of whitelisted origins.
+Returns the canister's list of trusted origins, meaning each front-end listed will not act maliciously with respect to this canister.
 
 ```
 icrc28_trusted_origins : () -> (record { trusted_origins : vec text });
 ```
 
+### icrc10_supported_standards
+
+An implementation of ICRC-28 MUST implement the method icrc10_supported_standards as put forth in 
+[ICRC-10](https://github.com/dfinity/ICRC/ICRCs/ICRC-10).
+
+The result of the call MUST always have at least the following entries:
+
+```
+record { name = "ICRC-28"; url = "https://github.com/dfinity/ICRC/ICRCs/ICRC-28"; }
```

Comment:
I'm really not a fan of the `url` entries here, since they do _not_ point anywhere sensible. Imho, at least the entry for ICRC-28 should be a correct URL, which would then be `https://github.com/dfinity/ICRC/tree/main/ICRCs/ICRC-28/ICRC-28.md`...

Reply:
updated (and created a PR for icrc28 placeholder in ICRCs repo